Source/WebCore/svg/SVGLengthContext.cpp

```diff
@@ -260,6 +264,28 @@ ExceptionOr<float> SVGLengthContext::convertValueFromEMSToUserUnits(float value)
     return value * style->computedFontSize();
 }
 
+ExceptionOr<float> SVGLengthContext::convertValueFromUserUnitsToREMS(float value) const
+{
+    auto* style = renderStyleForLengthResolving(protectedContext().get());
+    if (!style)
+        return Exception { ExceptionCode::NotSupportedError };
+
+    float fontSize = style->computedFontSize();
```

Comment:
This will be incorrect, we need to get the root element's font size, not the element's